pygmt/src/grdcut.py

```diff
@@ -14,14 +15,15 @@
     build_arg_list,
     data_kind,
     fmt_docstring,
+    tempfile_from_geojson,
     use_alias,
 )
 
 __doctest_skip__ = ["grdcut"]
 
 
 @fmt_docstring
-@use_alias(N="extend", S="circ_subregion", Z="z_subregion", f="coltypes")
+@use_alias(N="extend", S="circ_subregion", Z="z_subregion", f="coltypes", F="polygon")
```

Comment:
We have implemented a new alias system and will get rid of the `use_alias` decorator in the future.
```suggestion
@use_alias(N="extend", S="circ_subregion", Z="z_subregion", f="coltypes")
```